src/elements/content-sidebar/activity-feed/version/Version.scss

```diff
@@ -6,7 +6,8 @@
         align-items: center;
         justify-content: space-between;
         width: 100%;
-        margin-top: 10px;
+        margin-top: $bdl-grid-unit * 4;
+        margin-bottom: $bdl-grid-unit * 4;
```

Comment:
It's usually safer to apply padding/margin to only the top/left of items in a list. Does this create extra spacing if the version is the last item in the list, for example?